src/gtk/textctrl.cpp

```diff
@@ -323,6 +323,14 @@ au_apply_tag_callback(GtkTextBuffer *buffer,
 }
 }
 
+// Check if the style contains wxTE_PROCESS_TAB and update the given
+// GtkTextView accordingly.
+static void wxGtkSetAcceptsTab(GtkWidget* text, long style)
+{
+    gtk_text_view_set_accepts_tab(GTK_TEXT_VIEW(text),
+                                  (style & wxTE_PROCESS_TAB) ? TRUE : FALSE);
```

Comment:
I'd really like to see all uses of `TRUE`/`FALSE` in our code go away. It's C++, we have actual keywords for true and false, there is no reason not to use them. And this idiom, `condition ? true : false`, always makes me wonder if the writer doesn't understand what a boolean expression is.

Reply:
This was done in response to your request to avoid relying on implicit `bool -> gboolean` conversion in the original code... So now I'm totally confused as to what do you prefer.

I am fine with just using implicit conversion, i.e. writing `style & wxTE_PROCESS_TAB` here. I'm also fine with being explicit, as above. The one thing I'm not totally fine with is `(style & wxTE_PROCESS_TAB) != 0` because this is a weird middle ground which is neither implicit (because of comparison with 0) nor explicit (because there is still implicit `bool -> gboolean` conversion here). If this is really what you prefer, could you please explain what's your rationale for it?

Also, `TRUE` and `FALSE` are _not_ the same as `true` and `false`, they have different types (`gboolean`, a.k.a. `int`, for the former and `bool` for the latter). So it wouldn't be correct to use them interchangeably.

Reply:
I guess it wasn't clear what I meant originally. When you write `gtk_text_view_set_accepts_tab(GTK_TEXT_VIEW(m_text), style & wxTE_PROCESS_TAB);`, the type of the expression `style & wxTE_PROCESS_TAB` is `long`. This will be converted (truncated if `long` is 64-bit, a no-op if 32-bit) to `int` to match the function's parameter type. There is no implicit boolean conversion. You're literally passing either 0 or `wxTE_PROCESS_TAB` as the argument. And in theory, if `wxTE_PROCESS_TAB` represented a bit higher than 32 (I know it actually doesn't), the result would always be 0. I was just pointing out that I think it's preferable to explicitly convert that to a 0 or 1, and `(style & wxTE_PROCESS_TAB) != 0` would be the usual way to do that. 

And as for `true`/`false`, I don't think there is any reasonable situation where a compiler wouldn't convert them to a larger integral type without complaint.

Reply:
Compiler will convert, of course, but I just think it's more clear to use `TRUE`/`FALSE` when calling GTK+ functions. But I don't really feel strongly about it, so I'll change the code to use your preferred version.